app/javascript/components/applications/application_form/preferences/PreferenceAdditionalOptions.js

```diff
@@ -10,10 +10,10 @@ import RentBurdenedAssistedHousingFields from './RentBurdenedAssistedHousingFiel
 import DefaultPreferenceFields from './DefaultPreferenceFields'
 import AliceGriffithFields from './AliceGriffithFields'
 
-const PreferenceAdditionalOptions = ({i, householdMembers, listingPreferences, listingPreferenceID }) => {
+const PreferenceAdditionalOptions = ({i, shortFormPreference, householdMembers, listingPreferences, listingPreferenceID }) => {
   const preference = find(listingPreferences, { id: listingPreferenceID })
-  const propsFields = {i, householdMembers}
-  
+  const propsFields = {i, householdMembers, shortFormPreference}
+
   if (preference) {
     switch (preference.lottery_preference.name) {
       case 'Certificate of Preference (COP)':
```

Comment:
I'm not sure if I should go ahead and update all of the components below to expect shortFormPreferences too. Seems to work fine as is, but is most likely best practice to update all of them.

Alternatively, I could only pass shortFormPreference into the LiveWorkFields one

Reply:
@fddayan do you have any suggestions here?

Reply:
I think that passing the `shortFormPreference` is fine and makes sense.

Reply:
Should I update each of the preferences components declarations to include shortFormPreference then?

e.g. `const CertOfPreferenceFields = ({ i, householdMembers, shortFormPreference }) => {`

v.s. `const CertOfPreferenceFields = ({ i, householdMembers }) => {`

Reply:
sorry I misunderstood. no need to update the components to unpack `shortFormPreference`